generator-prisma-client/nextjs-starter-webpack/README.md

```
@@ -70,26 +70,26 @@ Note that `__YOUR_PRISMA_POSTGRES_CONNECTION_STRING__` is a placeholder value th
 
 Note that `__YOUR_PRISMA_POSTGRES_DIRECT_CONNECTION_STRING__` is a placeholder value that you need to replace with the values of your Prisma Postgres direct TCP connection string. The direct connection string has the following structure: `postgres://<username>:<password>@<host>:<port>/<database>`.
 
-### 3. Run a migration to create the database structure and seed the database
+### 3. Generate Prisma Client
 
-The [Prisma schema file](./prisma/schema.prisma) contains a single `Quotes` model and a `QuoteKind` enum. You can map this model to the database and create the corresponding `Quotes` table using the following command:
+Run:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)